### PR TITLE
Fix Azure backend integration tests user creation issue

### PR DESCRIPTION
**Pull request by MaciejTe**
Fix Azure backend integration tests user creation issue

Changelog: None

Signed-off-by: Maciej Tomczuk <maciej.tomczuk@northern.tech>

### CODE REVIEW

**Comment by MaciejTe** [comment]
@mender-test-bot start pipeline

File: backend-tests/tests/conftest.py [hunk 1]
```diff
@@ -65,3 +74,42 @@ def _get_endpoint_url(url):
...
+        "secretsecret",
+    )
+
+    user = create_user(user_name, password)
+    useradmm = ApiClient(useradm.URL_MGMT)
+    # log in user
+    response = useradmm.call("POST", useradm.URL_LOGIN, auth=(user.name, user.pwd))
+    assert response.status_code == 200
+    user.utoken = response.text
+    return user
```
Comment (lluiscampos) [inline backend-tests/tests/conftest.py:115]: In my opinion, these helpers should be in `testutils.common` and not in `conftest.py`. This file is meant to have only pytest fixtures, and it is included always when launching `pytest`.

Comment (MaciejTe) [reply backend-tests/tests/conftest.py]: You are right, I moved helper functions to `testutils.common`. Thank you!

**Review by tranchitella — APPROVED** [review]
It looks correct to me, can you please start the tests?

**Comment by tranchitella** [comment]
@mender-test-bot start pipeline

**Review by lluiscampos — APPROVED** [review]

File: backend-tests/tests/conftest.py [hunk 2]
```diff
@@ -11,7 +11,6 @@
 #    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #    See the License for the specific language governing permissions and
 #    limitations under the License.
-import urllib3
```
[the remark this answers is not in the document]
Comment (MaciejTe) [reply backend-tests/tests/conftest.py:14]: This import looks as if it's not used - `from requests.packages import urllib3` from line 24 is used in line 28

[the remark this answers is not in the document]
Comment (lluiscampos) [reply backend-tests/tests/conftest.py:14]: You are right, sorry

**Review by lluiscampos — APPROVED** [review]

**Pull request merged**